web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/fpoversikt/FpOversiktDtoTjeneste.java

```diff
@@ -69,8 +71,12 @@ public Sak hentSak(UUID behandlingId) {
     }
 
     private Optional<AktørId> oppgittAnnenPart(Fagsak fagsak) {
-        var førstegangsbehandling = behandlingRepository.finnSisteIkkeHenlagteBehandlingavAvBehandlingTypeFor(fagsak.getId(), BehandlingType.FØRSTEGANGSSØKNAD);
-        return personopplysningTjeneste.hentOppgittAnnenPartAktørId(førstegangsbehandling.orElseThrow().getId());
+        var førstegangsbehandling = behandlingRepository.finnSisteIkkeHenlagteBehandlingavAvBehandlingTypeFor(fagsak.getId(), BehandlingType.FØRSTEGANGSSØKNAD)
```

Comment:
Dette flyr ikke for førstegangsbehandlinger opprettet pga IM eller Infobrev .....

Reply:
Tenkte feil